src/Platforms/AbstractMySQLPlatform.php

```diff
@@ -439,8 +439,11 @@ public function getCreateTablesSQL(array $tables): array
                 Deprecation::trigger(
                     'doctrine/dbal',
                     'https://github.com/doctrine/dbal/pull/5414',
-                    'Relying on the DBAL not generating DDL for foreign keys on MySQL engines'
-                        . ' other than InnoDB is deprecated. Define foreign key constraints only if they are necessary.',
+                    <<<'DEPRECATION'
+                    Relying on the DBAL not generating DDL for foreign keys on
+                    MySQL engines other than InnoDB is deprecated.
+                    Define foreign key constraints only if they are necessary.
+                    DEPRECATION,
```

Comment:
If having the message on one line is important, I can use `str_replace` to remove newlines. Let me know.

Reply:
As I understand, the problem is that the newly added column makes the line longer than 120 characters. In this case, I'd break it down in two using concatenation. IMO, `str_replace()` would be an even worse hack than concatenation. And yes, I'd prefer to have the message on one line leaving the formatting to the presentation layer.